### PR TITLE
docs: improve Grids setup documentation

### PR DESCRIPTION
**Pull request by ncoden**
Add to documentation of the three grids informations about how to enable/disable it.

Closes https://github.com/zurb/foundation-sites/issues/10843

### CODE REVIEW

**Comment by colin-marshall** [comment]
@ncoden looks good! Do you want to also move `cell` to the beginning of all examples in this PR, or would you like me to create a separate PR for that?

File: docs/pages/grid.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ tags:
 
 ## Importing
 
-<div class="alert callout">
+<div class=" calloutalert">
```
Comment (colin-marshall) [inline docs/pages/grid.md]: You've got the space in the wrong place 😉

File: docs/pages/grid.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Start by adding an element with a class of `.row`. This will create a horizontal
   <a class="" data-open-video="1:07"><img src="{{root}}assets/img/icons/watch-video-icon.svg" class="video-icon" height="30" width="30" alt=""> Watch this part in video</a>
 </p>
 
-<div class="primary callout">
+<div class=" calloutprimary">
```
Comment (colin-marshall) [inline docs/pages/grid.md]: Same here

**Comment by ncoden** [comment]
I should definitely review myself more carefully.
Thanks @colin-marshall 👍

**Pull request merged**